File: app/src/main/java/chen/com/tools/IMChatView.java
package chen.com.tools;

import android.app.Activity;
import android.view.View;
import android.widget.EditText;

public interface IMChatView {

    Activity getActivity();

    void onKeyBoardChange(boolean isShow, int keyBoardHealth);

    View getEmojView();

    void showEmojView();

    void hideEmojView();

    void hideKeyBoard();

    void updateSoftInputMethod(int softInputMode);

    EditText getInputEditText();

    int getLayoutRes();

}

File: kotlin_test/src/main/java/chen/com/kotlin_test/IMChatView.kt
package chen.com.kotlin_test

import android.app.Activity
import android.view.View
import android.widget.EditText

interface IMChatView {
    fun getActivity():Activity?
    fun onKeyBoardChange(isShow: Boolean, keyBoardHealth: Int)
    fun getEmojView(): View?
    fun showEmojView()
    fun hideEmojView()
    fun hideKeyBoard()
    fun updateSoftInputMethod(softInputMode: Int)
    fun getInputEditText(): EditText?
    fun layoutRes(): Int
}